docker-compose.yml

```diff
@@ -54,6 +54,21 @@ qgis3:
   links:
     - db
 
+qgis3-trust:
+  image: camptocamp/qgis-server:3
+  environment:
+    QGIS_SERVER_LOG_LEVEL: 3
+    QGIS_OPTIONS_PATH: /root/
+    QGIS_SERVER_PARALLEL_RENDERING: "true"
+    QGIS_SERVER_MAX_THREADS: 8
```

Comment:
I doubt those 3 last variables reach QGIS. Apache is not forwarding automatically the environment variables to the FCGI daemon. You need to add stuff like that in the QGIS configuration:
https://github.com/camptocamp/docker-qgis-server/blob/master/server/runtime/etc/apache2/conf-enabled/qgis.conf#L13

Reply:
These lines are exactly the same as in qgis3 sections. I have just copy/paste the configuration.

The link you give is not reachable, so I don't know how to do so.

Reply:
This link is private :( but what about adding these variables in the Apache conf? I can do it if you agree.

Reply:
Sorry didn't remember that our QGIS image build was top secret :-(
I'm OK with adding those variables.

Reply:
@rldhont I will do this week and let you know as soon as it does.

Reply:
@yjacolin, let me do it. I have other things to sneak in there.